### PR TITLE
author:C3n7ral051nt4g3ncy | Adding 1 Site 

### PR DESCRIPTION
**Pull request by C3n7ral051nt4g3ncy**
- Added 1 website
- Chyoa 
- Porn | Erotic Reading/Stories
- User example: `https://chyoa.com/user/Joe`

### CODE REVIEW

File: web_accounts_list.json [hunk 1]
```diff
@@ -806,7 +806,17 @@
       "account_missing_code": "404",
       "known_accounts": ["test", "test2"],
       "category": "misc",
-      "valid": true
+      "valid": true     },
```
Comment (WebBreacher) [inline web_accounts_list.json:809]: Can you check on this line? Looks like there is a `^M` (Ctrl-M) in there. Please confirm or deny.

Comment (C3n7ral051nt4g3ncy) [reply web_accounts_list.json:809]: @WebBreacher

Answer= `deny`
I saw that yesterday after checking the PR, I thought I needed to make another commit to get rid of it, but I didn't find this anywhere.

Comment (C3n7ral051nt4g3ncy) [reply web_accounts_list.json:809]: @WebBreacher 
Found this:https://stackoverflow.com/questions/1822849/what-are-these-ms-that-keep-showing-up-in-my-files-in-emacs

**Comment by WebBreacher** [comment]
When I view your JSON file at https://github.com/WebBreacher/WhatsMyName/blob/75b3b330eab20cf5b0a490c87ad2fe660b57d6b1/web_accounts_list.json, I see the control-M (and an extra space in the "valid" param for the Chyoa site (see below). Can you go into the JSON file, remove that white space and then reformat it and resubmit?

image

**Comment by C3n7ral051nt4g3ncy** [comment]
> When I view your JSON file at https://github.com/WebBreacher/WhatsMyName/blob/75b3b330eab20cf5b0a490c87ad2fe660b57d6b1/web_accounts_list.json, I see the control-M (and an extra space in the "valid" param for the Chyoa site (see below). Can you go into the JSON file, remove that white space and then reformat it and resubmit?
> 
> image

Ok will do that right now.

**Comment by C3n7ral051nt4g3ncy** [comment]
@WebBreacher It seems to be only on yours.
I went on my Fork both on my PC and on GitHub, can't see anything at all:
https://github.com/C3n7ral051nt4g3ncy/WhatsMyName/edit/main/web_accounts_list.json

Screenshot below:
<br>

**Comment by C3n7ral051nt4g3ncy** [comment]
@WebBreacher How does it look now? I deleted up to the word `true`, skipped a line on purpose and then did another commit to put it back in its place. I can't see the `^M` on the last commit.

File: web_accounts_list.json [hunk 2]
```diff
@@ -808,6 +808,17 @@
...
+     {
+      "name" : "Chyoa",
+      "check_uri" : "https://chyoa.com/user/{account}",
+      "account_existence_code" : "200",
+      "account_existence_string" : "When I'm not reading erotica I like to read",
+      "account_missing_string" : "Sorry, I got distracted...",
+      "account_missing_code" : "404",
+      "known_accounts" : ["joe", "carlos01"],
+      "category" : "XXXPORNXXX",
+       "valid" : true
```
Comment (WebBreacher) [inline web_accounts_list.json:820]: Extra space in front of the `"valid" param. Please fix on your next PR. No need to resubmit this.

Comment (C3n7ral051nt4g3ncy) [reply web_accounts_list.json:820]: Copy. thanks

**Pull request merged**